playground/backend/internal/executors/executor.go

```diff
@@ -102,7 +104,9 @@ func (ex *Executor) Compile(ctx context.Context) *exec.Cmd {
 // Returns Cmd instance
 func (ex *Executor) Run(ctx context.Context) *exec.Cmd {
 	args := append(ex.runArgs.commandArgs, ex.runArgs.fileName)
-	cmd := exec.CommandContext(ctx, ex.runArgs.commandName, args...)
+	argsWithOptions := append(args, ex.runArgs.pipelineOptions)
+	argsWithOptions = utils.RemoveEmptyValue(argsWithOptions)
```

Comment:
Don't need to remove, please ensure that you don't add empty strings before

Reply:
it might come from the user side, i.e. --option1  option (2 whitespaces) and that valid command line way, however when we split to go cmd we should remove extra spaces :)

Reply:
Please replace with regex and move this part to the executor builder setup

Reply:
done